examples/se2-car.cpp

```diff
@@ -16,7 +16,7 @@ int main() {
   solver.setup(problem);
   solver.run(problem);
 
-  fmt::print("{}\n", fmt::streamed(solver.results_));
+  fmt::print("{}\n", solver.results_);
```

Comment:
This breaks with recent versions of fmtlib. A possible solution would be to specialise the required template class from fmtlib using `fmt::ostream_formatter`